common/tools/dev-tool/src/commands/customization/apply.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license
+
+import path from "path";
+
+import { resolveProject } from "../../util/resolveProject";
+import { createPrinter } from "../../util/printer";
+import { leafCommand } from "../../framework/command";
+import { makeCommandInfo } from "../../framework/command";
+
+import { customize } from "../../util/customization/customize";
+
+const log = createPrinter("apply-customization");
+
+export const commandInfo = makeCommandInfo("apply", "applies customizations to the SDK", {
+  sourceDirectory: {
```

Comment:
do we need the `Directory` part in the name? could it be anything else?

Reply:
I added to be explicit and avoid someone thinking they can pass a source file itself